benchmarks/Makefile

```diff
@@ -0,0 +1,40 @@
+MODE ?= debug
+top_srcdir ?= ..
+
+include $(top_srcdir)/Makefile.common
+
+########## LiveCode Script benchmark parameters
+
+BENCHMARKS_DIR ?= $(top_srcdir)/_benchmarks
+
+LCS_ENGINE ?= $(guess_engine)
+LCS_ENGINE_FLAGS ?= $(guess_engine_flags)
+
+LCS_LOG = _lcs_benchmarks.log
+LCS_BENCHMARKRUNNER ?= $(top_srcdir)/benchmarks/_benchmarkrunner.livecodescript
+
+LCS_CMD = $(LCS_ENGINE) $(LCS_ENGINE_FLAGS) $(LCS_BENCHMARKRUNNER) run
+
+
+################################################################
+# Top-level targets
+################################################################
+
+.DEFAULT: benchmark
+
+benchmark: lcs-benchmark
+
+clean:
+	-rm -rf $(BENCHMARKS_DIR) $(LCS_LOG)
+
+.PHONY: benchmark clean
+
+################################################################
+# Engine benchmarks
+################################################################
+
+lcs-benchmark: $(LCS_ENGINE)
+	@rm -f $(LCS_LOG)
+	@cmd="$(LCS_CMD)"; \
+	echo "$$cmd" $(_PRINT_RULE); \
+	$$cmd
```

Comment:
@livecodeali GNU make can behave weirdly if the makefile doesn't end with a linefeed.